tests/test_lcs.py

```diff
@@ -0,0 +1,126 @@
+# This file is part of ts_mtdome.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import unittest
+
+import numpy as np
+from lsst.ts import mtdome
+from lsst.ts.mtdome.mock_llc.lcs import (
+    NUM_LOUVERS,
+    NUM_MOTORS_PER_LOUVER,
+    POWER_PER_MOTOR,
+)
+
+logging.basicConfig(
+    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG
+)
+
+START_TAI = 10001.0
+
+
+class LcsTestCase(unittest.IsolatedAsyncioTestCase):
+    """A simple test class for testing some of the basic LCS commands. More
+    will be added as soon as the state machine for the LCS has been defined.
+    """
+
+    async def verify_lcs(
+        self, louver_ids: list[int], target_positions: list[float]
+    ) -> None:
+        """Utility method for verifying the positions of the louvers against
+        the provided IDs and target positions.
+
+        Parameters
+        ----------
+        louver_ids : `list` of `int`
+            A list with the IDs of the louvers to move.
+        target_positions : `list` of `float`
+            A list with the target positions of the louvers to move to.
+
+        Notes
+        -----
+        For each louver that will be moved, both an ID and a target position
+        needs to be given. It is assumed that the indices of the IDs and the
+        target positions are lined up.
```

Comment:
Please consider taking a dict of louver ID: target position as a single argument, instead of two lists that are *supposed* to be related to each other.

That makes the API easier to use correctly and harder to use incorrectly.
Furthermore I think it simplifies the code.

But if you prefer two lists then please at least check (assert) that they have the same length before doing anything else in this code.

Reply:
Great suggestion. I replaced it with a dict. Much cleaner this way!